server/controllers/profile.js

```diff
@@ -0,0 +1,53 @@
+const Profile = require("../models/ProfileModel");
+
+// create new profile using given arguments
+const addProfile = async (props) => {
+  let newProfile = await new Profile(props);
+  let profile = await newProfile.save();
+  return profile;
+};
+exports.createProfile = async (req, res, next) => {
+  try {
+    let response = await addProfile(req.body);
+    res.status(201).json(response);
+  } catch (err) {
+    next(err);
+  }
+};
+
+// updating existing profile using given id and new data
+const changeProfile = async (id, newData) => {
+  await Profile.findByIdAndUpdate(id, newData);
+};
+exports.updateProfile = (req, res, next) => {
+  try {
+    changeProfile(req.body.id, req.body);
+    res.status(201).json({ message: " your profile is updated" });
```

Comment:
Please update status codes to `200` throughout this file as code `201` is only used for resource creation